src/scales/scale.logarithmic.js

```diff
@@ -177,64 +177,95 @@ module.exports = function(Chart) {
 		getPixelForTick: function(index) {
 			return this.getPixelForValue(this.tickValues[index]);
 		},
+		/**
+		 * Returns the value of the first tick
+		 *
+		 * @Private
+		 * @param {Number} [value] - The minimum not zero value.
+		 * @return {Number} [firstTickValue] - The first tick value.
```

Comment:
Should be `@param {Number} value - The ...` since `value` is not optional (http://usejsdoc.org/tags-param.html)
Should be `@return {Number} The first ...`, without name and separator (http://usejsdoc.org/tags-returns.html)
Can you also move `@private` at the end and only lowercase, and I would remove the extra empty line:

```javascript
/**
 * Returns the value of the first tick.
 * @param {Number} value - The minimum not zero value.
 * @return {Number} The first tick value.
 * @private
 */
_getFirstTickValue: function(value) {
```

Similar to https://github.com/chartjs/Chart.js/blob/master/src/core/core.plugin.js#L92

Also we started to prefix private members by `_` ([for example](/chartjs/Chart.js/blob/master/src/helpers/helpers.core.js#L197)), though it's not yet consistent.